src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor(), new Random().nextInt(100));

```

Comment:
you should instantiate Random and ColorSupplier only once and reuse it multiple times. 

WIth current implementation you instantiates them each time the method is being invoked